qupath-gui-fx/src/main/resources/qupath/lib/gui/localization/qupath-gui-strings.properties

```diff
@@ -873,13 +873,13 @@ Prefs.Viewer.scalebarLineWidth.description = Adjust line thickness for the scale
 Prefs.Viewer.locationFontSize = Location text font size
 Prefs.Viewer.locationFontSize.description = Adjust font size for location text
 Prefs.Viewer.calibratedLocationString = Use calibrated location text
-Prefs.Viewer.calibratedLocationString.description = Show pixel locations on the viewer in �m where possible
+Prefs.Viewer.calibratedLocationString.description = Show pixel locations on the viewer in µm where possible
```

Comment:
This seems to display ok, but somehow git is picking up that it's different with a weird character.

Reply:
Yeah I don't know why it's happening